src/util/ui_orders.ts

```diff
@@ -58,9 +58,17 @@ export const mergeByPrice = (orders: OrderBookItem[]): OrderBookItem[] => {
                 };
             });
         })
-        .map(order => ({
-            side: order.side,
-            price: order.price,
-            size: order.size,
-        }));
+        .map(order => {
+            /* Avoids Typescript Compilation on filled field */
+            const orderToChange: any = order;
+            const newSize =
+                orderToChange.size && orderToChange.filled
```

Comment:
`orderToChange.filled` is not nullable, and by default it has 0, so this ternary doesn't seem necessary.